eva/udfs/new_udf_impl/base_udf_fns.py

```diff
@@ -0,0 +1,59 @@
+
+import numpy as np
+import pandas as pd
+import torch
+from facenet_pytorch import MTCNN
+import yolov5
+from eva.models.catalog.frame_info import FrameInfo
+from eva.models.catalog.properties import ColorSpace
+from eva.udfs.abstract.pytorch_abstract_udf import PytorchAbstractClassifierUDF
```

Comment:
I dislike the name of this file. Filenames should not be plural or contain abbreviations

Reply:
This is just a sample file and not the final one. A new file with proper naming conventions and coding practices will be created for the final integration into the eva system